MAUI/Cartesian-Charts/Annotation.md

```diff
@@ -0,0 +1,743 @@
+    };
+
+    chart.Annotations.Add(line);
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Line Annotation with linecap in MAUI Chart](Annotation_image/line_with_cap.png)
+
+## Vertical and Horizontal line annotation
+
+The [VerticalLineAnnotation]() and [HorizontalLineAnnotation]() are used to draw vertical and horizontal lines at specific points in the chart area.
+
+{% tabs %}
+
+ {% highlight xaml %}
+
+        <chart:SfCartesianChart>
+        ...
+        <chart:SfCartesianChart.Annotations>
+            <chart:VerticalLineAnnotation X1="2"/>
+            <chart:HorizontalLineAnnotation Y1="20"/>
+        </chart:SfCartesianChart.Annotations>
+
+        </chart:SfCartesianChart>
+
+{% endhighlight %} 
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+
+    var vertical = new VerticalLineAnnotation()
+    {
+        X1 = 2,
+    };
+
+    var horizontal = new HorizontalLineAnnotation()
+    {
+        Y1 = 20,
+    };
+
+    chart.Annotations.Add(vertical);
+    chart.Annotations.Add(horizontal);
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}  
+
+![Vertical and horizontal Line Annotation in MAUI Chart](Annotation_image/horizontal_vertical_annotation.png)
+
+## Displaying axis label for vertical and horizontal line annotations
+
+The [VerticalLineAnnotation]() and [HorizontalLineAnnotation]() display the axis labels where the line is placed. This feature can be enabled by setting the [ShowAxisLabel]() property to true, as shown in the code snippet below:
+
+{% tabs %}
+
+ {% highlight xaml %}
+
+        <chart:SfCartesianChart>
+        ...
+        <chart:SfCartesianChart.Annotations>
+            <chart:VerticalLineAnnotation X1="2" ShowAxisLabel="True"/>
+            <chart:HorizontalLineAnnotation Y1="20" ShowAxisLabel="True"/>
+        </chart:SfCartesianChart.Annotations>
+
+        </chart:SfCartesianChart>
+
+{% endhighlight %} 
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+
+    var vertical = new VerticalLineAnnotation()
+    {
+        X1 = 2,
+        ShowAxisLabel= true
+    };
+
+    var horizontal = new HorizontalLineAnnotation()
+    {
+        Y1 = 20,
+        ShowAxisLabel= true
+    };
+
+    chart.Annotations.Add(vertical);
+    chart.Annotations.Add(horizontal);
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}  
+
+![Vertical and horizontal Line Annotation with axis label in MAUI Chart](Annotation_image/axis_label.png)
+
+## Customizing axis label
+
+The default appearance of the axis label can also be customized using the [AxisLabelStyle]() property. The following styles are used to customize the axis label:
+
+The [ChartLabelStyle]() property allows you to customize the axis labels by changing the font family, font size, font attributes, and text color. It is the default style for the [AxisLabelStyle]() property. The following properties can be used to customize the axis label:
+
```

Comment:
@NitheeshkumarThangaraj  , I have provided the suggestion to use sentences as MAUI customization categories, like plot band and axis level, which we already discussed during the scrum call. Why is this not addressed?

Reply:
@SaiyathAliFathima , addressed and updated the content